### PR TITLE
updating toolkit docs with people picker component

### PR DESCRIPTION
**Pull request by vogtn**

### CODE REVIEW

File: concepts/toolkit/components/people-picker.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: "People-Picker component"
+description: "You can use the mgt-people-picker web component to search for and hold a requested number of people via Microsoft Graph."
+localization_priority: Normal
+author: vogtn
+---
+
+# People-Picker component
+
+You can use the `mgt-people-picker` web component to search for and hold a requested number of people via Microsoft Graph. By default, the component will search for all people; you can also define a group property to further filter the results.
```
Comment (Lauragra) [inline concepts/toolkit/components/people-picker.md]: What do we mean by "hold a requested number of people?"

Comment (vogtn) [reply concepts/toolkit/components/people-picker.md]: Hi @Lauragra thanks for looking. Is " hold a requested number of selected people" or "chosen people" better terminology?

One of the main features of the component is for the user to be able to choose or pick people, the selected people are then rendered in a stylized list.

Comment (Lauragra) [reply concepts/toolkit/components/people-picker.md]: What if we revise as: "You can use the mgt-people-picker web component to search for a specified number of people and render the list of results via Microsoft Graph."?

Comment (vogtn) [reply concepts/toolkit/components/people-picker.md]: Thanks @Lauragra for the suggestion! I've updated in the latest commit. 

https://github.com/microsoftgraph/microsoft-graph-docs/pull/5453/commits/63fc97eaace2a19dabac6ac9ade9cfec663e7ed3

Comment (Lauragra) [reply concepts/toolkit/components/people-picker.md]: Looks good - merging now!

**Pull request merged**